content/events.md

```diff
@@ -0,0 +1,20 @@
+@title Events
+@navbar
+
+# Events
+The Polympics will host ten event contests, each having a set of commissioners (directors).  All of the commissioners together are Polympic Committee. 
+
+Players register through the Polympics website, become a member of the Polympics discord server, and join a team based on nationality. Teams will send players to compete in events. Some events allow multiple entries per team, others are limited to one representative (or pair of representatives) per team. See event rules for more details. Some events are mobile only while others allow for Steam or mobile users.
+
+## Events:
+- [Co-op Monument Rush](events/monument-rush)
+- [1v1 Might](events/1v1-might)
+- [2v2 Team Marathon](events/2v2-team-marathon)
+- [FFA Relay](events/ffa-relay)
+- [1v1 Bullet](events/1v1-bullet)
+- [1v1 Glory/Wicked Bender](events/1v1-glory-wicked-bender)
+- [Art Contest](events/art-contest)
+- [Skyscraper](events/skyscraper)
+- [Perfect 10](events/perfect-10)
+- [Apocalypse (1v3)](events/apocalypse-1v3)
```

Comment:
Do we really need the same list on three pages? (yes is a valid answer)

Reply:
I have no idea. I leave it to you.

Honestly, I am trying to help, but I feel a bit lost. I defer to what ever you think is best.

Reply:
Well I can see how it would be helpful to link to this list specifically, and I don't forsee us needing to update the list much, so I'll go ahead and merge it.